src/autopas/containers/cellPairTraversals/SlicedTraversal.h

```diff
@@ -83,14 +83,23 @@ inline void SlicedTraversal<ParticleCell, PairwiseFunctor, useSoA, useNewton3>::
 
   // split domain across its longest dimension
 
-  auto numSlices = autopas_get_max_threads();
-  // find dimension with most cells
+  auto numSlices = (size_t)autopas_get_max_threads();
+  auto minSliceThickness = this->_cellsPerDimension[_dimsPerLength[0]] / numSlices;
+  if (minSliceThickness < 2) {
+    AutoPasLog(debug, "Sliced travesal only using {} threads because number of cells is too small.", numSlices)
+        minSliceThickness = 2;
+    numSlices = this->_cellsPerDimension[_dimsPerLength[0]] / minSliceThickness;
+  }
+
   _sliceThickness.clear();
-  _sliceThickness.insert(_sliceThickness.begin(), numSlices, this->_cellsPerDimension[_dimsPerLength[0]] / numSlices);
+
+  // abort if domain is too small -> cleared _sliceThickness array indicates non applicability
+  if (numSlices < 1) return;
+
+  _sliceThickness.insert(_sliceThickness.begin(), numSlices, minSliceThickness);
   auto rest = this->_cellsPerDimension[_dimsPerLength[0]] - _sliceThickness[0] * numSlices;
   for (size_t i = 0; i < rest; ++i) ++_sliceThickness[i];
-  // decreases last _sliceThickness by one to account for the way we handle base
-  // cells
+  // decreases last _sliceThickness by one to account for the way we handle base cells
   --*--_sliceThickness.end();
```

Comment:
why does github show --*-- here?

Reply:
because it is exactly what the comment above says.
last _sliceThickness -> --_sliceThickness.end()
dereference -> *
decrese by one -> --
:innocent:

Reply:
brackets, please!

Reply:
and -1 instead of --. at least for the end thing